docs/unattend.xml

```diff
@@ -18,8 +18,6 @@
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
          <!-- Windows host computer name -->
          <ComputerName>windows-host</ComputerName>
-         <!-- Windows product key -->
-         <ProductKey>_WINDOWS_PRODUCT_KEY_</ProductKey>
```

Comment:
This is confusing given you add it in one commit and then remove it here. Do you need to add it in the first place?